### PR TITLE
pferd: 3.4.3 -> 3.5.0

### PR DESCRIPTION
**Pull request by 0xbe7a**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/pferd/default.nix [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
 python3Packages.buildPythonApplication rec {
   pname = "pferd";
-  version = "3.4.3";
+  version = "3.5.0";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "Garmelon";
     repo = "PFERD";
-    rev = "refs/tags/v${version}";
```
Comment (dotlambda) [inline pkgs/tools/misc/pferd/default.nix:14]: Why change that?

File: pkgs/tools/misc/pferd/default.nix [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
-    sha256 = "sha256-m9u4CLvyF2n61JQyxEKlVzWZCUhzVEVqpSZMNDZXCAo=";
+    rev = "v${version}";
+    sha256 = "1mi3ql6ryfgg31airq3kgz5a5lbprszddm0gggkrkwiz2cljlqw8";
```
Comment (dotlambda) [inline pkgs/tools/misc/pferd/default.nix:15]: Please use an SRI hash.
```suggestion
    hash = "1mi3ql6ryfgg31airq3kgz5a5lbprszddm0gggkrkwiz2cljlqw8";
```

File: pkgs/tools/misc/pferd/default.nix (hunk 1 above)
Comment (0xbe7a) [reply pkgs/tools/misc/pferd/default.nix:14]: my bad, lost in rebase

**Comment by 0xbe7a** [comment]
@dotlambda can you review again?

**Comment by 0xbe7a** [comment]
@SuperSandro2000 this is ready to review & merge 😀

**Pull request merged**